tests/pkcli/ci1_data/1.in/package_data/x.py

```diff
@@ -1 +1 @@
-print("this should be ingore since package_data")
+print("this should be ignore since package_data")
```

Comment:
So now the test directory needs to be structured like a real repo with a setup.py (can be blank) and package_data under some package name. The same is true of the rest of the files.

Reply:
@robnagler made changes with the above dir structure. LMK what you think

Reply:
This is not a repo structure. Look at a repo and see how the above differs from what is there.

Reply:
Ok so what about this?
```
 1.in/
    setup.py
    in/
        has_prints.py
        package_data/
            x.py
    tests/
        test_data/
           x.py
   
```

Reply:
x.py could be renamed "not_checked.py" or something so the file names reflect what the test is supposed to be doing.